htdocs/scss/pages/rename.scss

```diff
@@ -25,15 +20,6 @@
     padding: 0.2em 0;
 }
 
-ul.error-list {
-    margin-left: 2em;
-    margin-bottom: 2em;
-}
-
-ul.error-list li { 
-    list-style: disc outside;
-}
-
-p.detail {
-    margin-left: 2em !important;
+#content p.note {
+    font-style: italic; font-size: 80%;
```

Comment:
I believe the style is to have a newline after semicolons, so they're on separate lines?